script.js

```diff
@@ -0,0 +1,52 @@
+let tasks = [];
+let taskId = 0;
+
+function openForm() {
+    document.getElementById("taskForm").style.display = "flex";
```

Comment:
Move the dom element declaration to another element.js file, declare it as a const and then export it. Finally in script file import it to use.
For style it's better to use style.css. you can add a class and by JS, you can add or remove it from the DOM.